http3/server.go

```diff
@@ -230,6 +230,35 @@ func (s *Server) maxHeaderBytes() uint64 {
 }
 
 func (s *Server) handleRequest(sess quic.Session, str quic.Stream, decoder *qpack.Decoder, onFrameError func()) requestError {
+
+	var (
+		wholeReadDeadline time.Time
+		hdrDeadline       time.Time
+
+		setReadDeadline bool
+	)
+	t0 := time.Now()
+	// set read timeout
+	if d := s.ReadTimeout; d != 0 {
+		wholeReadDeadline = t0.Add(d)
+		err := str.SetReadDeadline(wholeReadDeadline)
+		if err != nil {
+			return newStreamError(errorSettingsError, err)
```

Comment:
This should be an internal error. Or maybe we need a timer here instead of a deadline (see comment)?

Reply:
Yes we can remove it and let quic handle ReadTimeout by setting MaxIdleTimeout. Make sense.

Reply:
If we want to remove it we have to disable QUIC idleTimeout timer after parsing header. Let's keep it as it is.